### PR TITLE
microprofile support in sync with jdbc booster support

### PR DESCRIPTION
**Pull request by ajm01**
this is a re-do of the microprofile support from just prior to scott's december comments in PR https://github.com/OpenLiberty/boost/pull/222

Once this level of microprofile support is in maste, will address the additional comment changes made in that PR.

### CODE REVIEW

File: boost-maven/boost-maven-plugin/src/it/test-mpHealth-1.0/src/test/java/it/io/openliberty/guides/health/HealthTestIT.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import org.junit.After;
+import org.junit.Test;
+
+public class HealthTestIT {
+
+    private JsonArray servicesStates;
+    private static HashMap<String, String> dataWhenServicesUP;
+    private static HashMap<String, String> dataWhenInventoryDown;
+
+    static {
```
Comment (awisniew90) [inline boost-maven/boost-maven-plugin/src/it/test-mpHealth-1.0/src/test/java/it/io/openliberty/guides/health/HealthTestIT.java]: Should this method just be an init() or setup() with the `@BeforeClass` annotation?

File: boost-common/src/main/java/io/openliberty/boost/common/config/BoosterPackConfigurator.java [hunk 2]
```diff
@@ -22,6 +22,8 @@
 		
 	protected String EE_7_VERSION = "0.1-SNAPSHOT";
 	protected String EE_8_VERSION = "0.2-SNAPSHOT";
+    protected String MP_20_VERSION = "0.2-SNAPSHOT";
+
```
Comment (scottkurz) [inline boost-common/src/main/java/io/openliberty/boost/common/config/BoosterPackConfigurator.java]: Code formatting

File: boost-maven/boost-boms/boost-mp20-apis-bom/pom.xml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	        <dependency>
+	            <groupId>javax.json</groupId>
+	            <artifactId>javax.json-api</artifactId>
+	            <version>${jsonp-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.annotation</groupId>
+	            <artifactId>javax.annotation-api</artifactId>
+	            <version>${annotation-version}</version>
+	        </dependency>
```
Comment (scottkurz) [inline boost-maven/boost-boms/boost-mp20-apis-bom/pom.xml]: Unlike, say, cdi-api, this dependency is not used by one of our boosters.    I wonder if we've thought through the pros/cons of bringing it into the BOM, then?

File: boost-maven/boost-boms/pom.xml [hunk 4]
```diff
@@ -12,9 +11,10 @@
     <parent>
         <groupId>io.openliberty.boost</groupId>
         <artifactId>boost-maven-parent</artifactId>
-        <version>0.1.3-SNAPSHOT</version>
+        <version>0.1.2-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (ajm01) [reply boost-maven/boost-boms/pom.xml]: fixed

**Comment by scottkurz** [comment]
Please FUP on these comments along with the comments in #222.  Merging

**Comment by scottkurz** [comment]
The checks aren't here.   That's because this PR was wrongly closed w/o merging, then re-opened to preserve its comments and to avoid creating yet another PR for this issue (we already have #222).

Instead the checks are in a parallel, equivalent PR on my personal fork here:
https://github.com/scottkurz/boost-maven/pull/6/checks

Merging now.

**Pull request merged**